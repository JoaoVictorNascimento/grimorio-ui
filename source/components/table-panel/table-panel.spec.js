import '../../../internals/test/helper';

import TablePanel from './table-panel-component';

/** @test {TablePanel} */
describe('TablePanel component', () => {
/** @test {TablePanel#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <TablePanel />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
