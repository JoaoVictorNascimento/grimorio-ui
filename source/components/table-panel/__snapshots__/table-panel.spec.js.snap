// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TablePanel component #render when renders correctly 1`] = `
"<Panel size=\\"no-padding\\" className=\\"wrap\\" footer={false} title={false} loading={false} accordion={false} open={true} highlight=\\"shadow\\">
  <div className=\\"header\\" />
  <Table type=\\"panel\\" schema={{...}} data={{...}} layout=\\"auto\\" notFoundMessage=\\"Nenhum resultado encontrado :(\\" loadingMessage=\\"Loading...\\" isSticky={false} />
</Panel>"
`;
